### DISCUSSION TITLE
fyne.canvasObject

### DISCUSSION THREAD

**Discussion by trodger88** [fyne-io/fyne#4898] [General] [answered]
When running the AppTab example in the docs , I get 
the error that the AppTab is not a fyne.canvasObject
tabs := container.NewAppTabs(
		container.NewTabItem("Tab 1", widget.NewLabel("Hello")),
		container.NewTabItem("Tab 2", widget.NewLabel("World!")),
	)

	tabs.SetTabLocation(container.TabLocationTop)
	myWindow.SetContent(tabs)

error .CanvasObject value in argument to myWindow.SetContent: *container.AppTabs does not implement "fyne.io/fyne".CanvasObject (wrong type for method MinSize)
		have MinSize() "fyne.io/fyne/v2".Size
		want MinSize() "fyne.io/fyne".Size

**Comment by dweymouth** [comment, marked as answer]
It looks like you have a mix of fyne v1 and v2 imports in your code. (IDEs can sometimes mess this up when doing auto-imports). Please make sure all your fyne imports are /v2
